src/content/en/2022/capabilities.md

```diff
@@ -18,5 +18,110 @@ featured_stat_3: TODO
 featured_stat_label_3: TODO
 unedited: true
 ---
+<!-- markdownlint-disable MD024 -->
```

Comment:
Why do we need this?

Reply:
For every API discussed, I have a section discussing growth/usage. It's an h3, but then that means each API has a heading of `### Growth`.  I'll rename them so header links will work correctly, so it's more or less a temp thing as I get the chapter added.

Reply:
Cool. Just checking you knew they'd need to be unqiue for the links.

BTW there's a nice [Google Docs extension to convert to Markdown](https://workspace.google.com/marketplace/app/docs_to_markdown/700168918607) so you don't need to do it manually bit by bit if that helps? Doesn't do the figures (and need to delete the code where it tries it's best but fails) but works nicely for the rest.